src/components/Sign-In.js

```diff
@@ -0,0 +1,54 @@
+import React, {Component} from 'react'
+// import {Route, Switch, Redirect} from 'react-router-dom'
+
+class SignIn extends Component {
+  constructor(props){
+    super(props);
+    this.state = {
+      email: '',
+      password: ''
+    }
+  }
+  updateState = (event) => {
+    this.setState({ [event.target.name]: event.target.value })
+  }
+
+  evaluateUser = (event) => {
+    event.preventDefault()
+    if (this.state.email.toLowerCase() === 'lucy@turing.io' && this.state.password === 'password1'){
+      const data = {
+        email: 'lucy@turing.io',
+        password: 'password1'
+      }
+      fetch('https://rancid-tomatillos.herokuapp.com/api/v2/login', {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json'
+        },
+        body: JSON.stringify(data)
+      })
+      .then(response => response.json())
```

Comment:
Let's add a response.ok check on this one on our next pass through just in case the server decides to return a 4-- range error that won't get caught with catch.